package proto_ff;

import "yd_fieldoptions.proto";


message godRelicsgodRelics
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 functionId = 2[(yd_fieldoptions.field_cname) = "功能开放id"];
	optional int32 group = 3[(yd_fieldoptions.field_cname) = "条件组ID"];
	optional int32 skillID = 4[(yd_fieldoptions.field_cname) = "激活技能"];
}


message Sheet_godRelicsgodRelics
{
	repeated godRelicsgodRelics godRelicsgodRelics_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message godRelicscondition
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 group = 2[(yd_fieldoptions.field_cname) = "条件组id"];
	optional int32 conditon = 3[(yd_fieldoptions.field_cname) = "条件类型"];
	optional string parama = 4[(yd_fieldoptions.field_cname) = "条件参数", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 count = 5[(yd_fieldoptions.field_cname) = "完成次数"];
	optional int32 rewardItem = 6[(yd_fieldoptions.field_cname) = "奖励道具"];
	optional int32 rewardNumber = 7[(yd_fieldoptions.field_cname) = "奖励道具数量"];
}


message Sheet_godRelicscondition
{
	repeated godRelicscondition godRelicscondition_List = 1[(yd_fieldoptions.field_arysize)=20];
}
